doc/releases/quincy.rst

```diff
@@ -0,0 +1,3656 @@
+======
+Quincy
+======
+
+Quincy is the 17th stable release of Ceph.  It is named after Squidward
+Quincy Tentacles from Spongebob Squarepants.
+
+v17.2.0 Quincy
+==============
+
+This is the first stable release of Ceph Quincy.
+
+Major Changes from Pacific
+--------------------------
+
+General
+~~~~~~~
+
+* Filestore has been deprecated in Quincy. BlueStore is Ceph's default object
+  store.
+
+* The `ceph-mgr-modules-core` debian package no longer recommends
+  `ceph-mgr-rook`. `ceph-mgr-rook` depends on `python3-numpy`, which
+  cannot be imported in different Python sub-interpreters multiple times
+  when the version of `python3-numpy` is older than 1.19. Because
+  `apt-get` installs the `Recommends` packages by default, `ceph-mgr-rook`
+  was always installed along with the `ceph-mgr` debian package as an
+  indirect dependency. If your workflow depends on this behavior, you
+  might want to install `ceph-mgr-rook` separately.
+
+* The ``device_health_metrics`` pool has been renamed ``.mgr``. It is now
+  used as a common store for all ``ceph-mgr`` modules.
+
+* The ``ceph pg dump`` command now prints three additional columns:
+  `LAST_SCRUB_DURATION` shows the duration (in seconds) of the last completed
+  scrub;
+  `SCRUB_SCHEDULING` conveys whether a PG is scheduled to be scrubbed at a
+  specified time, whether it is queued for scrubbing, or whether it is being
+  scrubbed;
+  `OBJECTS_SCRUBBED` shows the number of objects scrubbed in a PG after a
+  scrub begins.
+
+* A health warning is now reported if the ``require-osd-release`` flag
+  is not set to the appropriate release after a cluster upgrade.
+
+* LevelDB support has been removed. ``WITH_LEVELDB`` is no longer a supported
+  build option.
```

Comment:
What is the expected behaviour for old clusters with leveldb mon stores -- will rocks load these or does the admin need to recreate the mon?

Reply:
Good catch, I was also wondering what "support" means.  "You're on your own" vs "It won't work" - from the wording above I suspect that it's the latter and it would be good to clarify.

Reply:
The mon store would need to be migrated to rocksdb before upgrading to quincy. We can make this note clearer and add additional information in the "Upgrading from Octopus or Pacific" section https://github.com/ceph/ceph/pull/45048/files#diff-9002bf2431e8ce44f07242373c708feaaa05aafd1d42569f6cb16fd1d0e40942R227. That should be enough? I don't think we want to add a health warning at this point.

Reply:
emphasized here and in upgrade notes